.github/workflows/docker.yml

```diff
@@ -71,4 +90,14 @@ jobs:
         if: github.event_name == 'push'
         run: |
           echo "${{ secrets.DOCKER_BOT_TOKEN }}" | docker login -u dukeroboticsbot --password-stdin
-          docker push ${IMAGE_NAME}
\ No newline at end of file
+          docker push ${IMAGE_NAME}
+
+  cleanup:
```

Comment:
it could also make sense to fold this step into the simulation job. not sure we get any useful information from a separate job and the artefact is simulation related.